genienlp/validate.py

```diff
@@ -410,7 +410,7 @@ def validate(task, val_iter, model, numericalizer, args, num_print=10):
             output.predictions, output.answers, metrics_to_return, args.reduce_metrics, model.tgt_lang
         )
 
-        results = {'beam search': output.predictions, 'answer': output.answers, 'context': output.contexts}
+        results = {'model prediction': output.predictions, 'answer': output.answers, 'context': output.contexts}
```

Comment:
Can we use "prediction" instead to be consistent with other keys?

Reply:
I think "prediction" and "answer" are vague to first time users, as answer in some other code bases means the answer the model gives. So I changed them to "model prediction" and "gold answer".

Reply:
yeah, this is another one of those things we need to standardize throughout the code. We have "beam search" here and "greedy" in metrics.py!
looks good for this PR, but should we go with "pred" and "gold" everywhere?